services/brig/test/integration/API/Metrics.hs

```diff
@@ -31,25 +28,6 @@ testPrometheusMetrics brig = do
         -- Should contain the request duration metric in its output
         const (Just "TYPE http_request_duration_seconds histogram") =~= responseBody
 
-testMonitoringEndpoint :: Brig -> Http ()
-testMonitoringEndpoint brig = do
-    let p1 = "/self"
-        p2 uid = "/users/" <> uid <> "/clients"
-
-    uid <- userId <$> randomUser brig
-    uid' <- userId <$> randomUser brig
-    _ <- get (brig . path p1 . zAuthAccess uid "conn" . expect2xx)
-    _ <- get (brig . path (p2 $ toByteString' uid) . zAuthAccess uid "conn" . expect2xx)
-    _ <- get (brig . path (p2 $ toByteString' uid') . zAuthAccess uid "conn" . expect2xx)
-
-    get (brig . path "i/metrics") !!! do
-        -- GET /self was called once
-        const (Just "http_request_duration_seconds_count{handler=\"/self\",method=\"GET\",status_code=\"200\"} 1") =~= responseBody
```

Comment:
i'm tempted to suggest that we test these numbers as lower bounds at least?  or twice, once in line 38 with lower bound 0, recording the actual number, and once here, using the recorded number plus 1 as the new lower bound?

my tempation is even stronger since the FUTUREWORK below that was inspired by this test, and since the original code in this test *did* work.

Reply:
Yes, good point. I re-implemented the test.